packages/infra/lib/secrets-stack.ts

```diff
@@ -79,6 +79,13 @@ export class SecretsStack extends Stack {
     }
 
     if (!isSandbox(props.config)) {
+      if (props.config.surescripts) {
+        for (const secretName of Object.values(props.config.surescripts.secrets)) {
+          const secret = makeSecret(secretName);
+          logSecretInfo(this, secret, secretName);
+        }
+      }
```

Comment:
i think we've been leaning towards dropping the isSandbox checks and just relying on whether 'surescripts' field is in a given config. Ship'n'fix pl0x